src/commands/Social/autorole.ts

```diff
@@ -71,58 +71,58 @@ export default class extends SkyraCommand {
 	}
 
 	public async add(message: GuildMessage, [role, points]: [Role, number]) {
-		const language = await message.guild.writeSettings((settings) => {
-			const language = settings.getLanguage();
+		const t = await message.guild.writeSettings((settings) => {
+			const t = settings.getLanguage();
 			const roles = settings[GuildSettings.Roles.Auto];
 
 			if (roles.length && roles.some((entry) => entry.id === role.id)) {
-				throw language.get(LanguageKeys.Commands.Social.AutoRoleUpdateConfigured);
+				throw t(LanguageKeys.Commands.Social.AutoRoleUpdateConfigured);
 			}
 
 			const sorted = [...roles, { id: role.id, points }].sort(SORT);
 
 			settings[GuildSettings.Roles.Auto] = sorted;
-			return language;
+			return t;
 		});
 
-		return message.send(language.get(LanguageKeys.Commands.Social.AutoRoleAdd, { role, points }));
+		return message.send(t(LanguageKeys.Commands.Social.AutoRoleAdd, { role, points }));
 	}
 
 	public async remove(message: GuildMessage, [role]: [Role]) {
-		const [roleEntry, language] = await message.guild.writeSettings((settings) => {
-			const language = settings.getLanguage();
+		const [roleEntry, t] = await message.guild.writeSettings((settings) => {
+			const t = settings.getLanguage();
 			const roles = settings[GuildSettings.Roles.Auto];
 			const roleIndex = roles.findIndex((entry) => entry.id === role.id);
 
 			if (roleIndex === -1) {
-				throw language.get(LanguageKeys.Commands.Social.AutoRoleUpdateConfigured);
+				throw t(LanguageKeys.Commands.Social.AutoRoleUpdateConfigured);
```

Comment:
Technically this isn't the right message for a missing entry. Use the same one as in update